thrall/app/controllers/ReaperController.scala

```diff
@@ -119,14 +119,15 @@ class ReaperController(
         )
       ))
     } yield {
-      if(idsSoftDeletedInES.isEmpty){
+      if(esIds.isEmpty){
```

Comment:
Not a big thing, but I was a bit surprised that we log here if no images were found to be deleted, but we don't log if images were found but failed to be deleted. Until I got to `softDeleteNextBatchOfImages` and found that it logs failures. I wonder whether these two bits of logging should be closer together, probably in `softDeleteNextBatchOfImages`?

Reply:
done - see https://github.com/guardian/grid/pull/4151/commits/9bf496b29e33779b3f61b8d2e663c9e617809ccd

Reply:
Thanks!